shared/src/main/scala/Elements.scala

```diff
@@ -1270,6 +1304,29 @@ object Elements:
       ctx.wrap
     }
 
+    val wrapSingleton = addElem(
+      Monad,
+      "w",
+      "Wrap Singleton",
+      List("wrap-singleton"),
+      "a -> [a]"
+    ) { case a =>
+      VList(a)
+    }
```

Comment:
```suggestion
    val wrapSingleton = addFull(
      Monad,
      "w",
      "Wrap Singleton",
      List("wrap-singleton"),
      false,
      "a -> [a]"
    ) { a => VList(a) }
```
Since it works on all elements. Not committing this suggestion because I don't know if we might need it to fail on some cases in the future (also, no idea if this compiles)